### PR TITLE
Update modelsbuildersettings.md

### PR DESCRIPTION
**Pull request by Jette94**
Added dot to "Let's go through them one by one" 

Also added a comment on best practices for Models modes when working with Models builder, as this can affect performance on the website. A general thing on all versions when it comes to models builder.

## Description

_What did you add/update/change?_

## Type of suggestion

* [ ] Typo/grammar fix
* [ ] Updated outdated content
* [ ] New content
* [ ] Updates related to a new version
* [ ] Other

## Deadline (if relevant)

_When should the content be published?_

### CODE REVIEW

File: 14/umbraco-cms/reference/configuration/modelsbuildersettings.md [hunk 1]
```diff
@@ -32,6 +32,10 @@ Specifies how the models builder will generate models and when to generate them.
 * `SourceCodeManual` - Models will be generated as `.cs` files whenever a user clicks the "Generate models" button on the models builder dashboard - however, the models will not be compiled and loaded into memory dynamically. This means that models are available to edit within the project. The project needs to be recompiled and restarted for the new models, or model changes, to take effect.
 * `SourceCodeAuto` - This mode behaves the same as `SourceCodeManual` with one difference, the generation of models happens automatically every time a content type change occurs.
 
+{% hint style="code" %}
+As a best practice of using Models Builder, this setting should be used for all appsettings.json files. Then on the local appsettings.json config transform (which is a copy of appsettings.Development.json) change the models mode to SourceCodeManual or SourceCodeAuto.
```
Comment (Jette94) [inline 14/umbraco-cms/reference/configuration/modelsbuildersettings.md:36]:
```suggestion
As a best practice of using Models Builder, the "Nothing" setting should be used for all appsettings.json files. Then on the local appsettings.json config transform (which is a copy of appsettings.Development.json) change the models mode to "SourceCodeManual" or "SourceCodeAuto".
```

File: 14/umbraco-cms/reference/configuration/modelsbuildersettings.md [hunk 2]
```diff
@@ -32,6 +32,10 @@ Specifies how the models builder will generate models and when to generate them.
 * `SourceCodeManual` - Models will be generated as `.cs` files whenever a user clicks the "Generate models" button on the models builder dashboard - however, the models will not be compiled and loaded into memory dynamically. This means that models are available to edit within the project. The project needs to be recompiled and restarted for the new models, or model changes, to take effect.
 * `SourceCodeAuto` - This mode behaves the same as `SourceCodeManual` with one difference, the generation of models happens automatically every time a content type change occurs.
 
+{% hint style="code" %}
```
Comment (sofietoft) [inline 14/umbraco-cms/reference/configuration/modelsbuildersettings.md:35]:
```suggestion
{% hint style="info" %}
```

File: 14/umbraco-cms/reference/configuration/modelsbuildersettings.md [hunk 3]
```diff
@@ -32,6 +32,10 @@ Specifies how the models builder will generate models and when to generate them.
...
+As a best practice of using Models Builder, the "Nothing" setting should be used for all appsettings.json files. Then on the local appsettings.json config transform (which is a copy of appsettings.Development.json) change the models mode to "SourceCodeManual" or "SourceCodeAuto".
```
Comment (sofietoft) [inline 14/umbraco-cms/reference/configuration/modelsbuildersettings.md:36]:
```suggestion
when using Models Builder it is best practice to use the "Nothing" setting for all `appsettings.json` files. In the local `appsettings.json` config transform (which is a copy of appsettings.Development.json) the models mode can be changed to "SourceCodeManual" or "SourceCodeAuto" if needed.
```

**Review by sofietoft — COMMENTED** [review]
Thanks for the PR @Jette94 !

I've made a couple of suggestions.
Also, just for clarification: 
Is it a config transform file of the appsettings.json file that you have locally, that needs to be changed? Is it already there, or do you need to make the copy yourself?

File: 14/umbraco-cms/reference/configuration/modelsbuildersettings.md (hunk 2 above)
Comment (Jette94) [reply 14/umbraco-cms/reference/configuration/modelsbuildersettings.md:35]: Hi Sofie! 

The local one is normally a copy you can make yourself from Development - after that, you can rename it.

Comment (sofietoft) [reply 14/umbraco-cms/reference/configuration/modelsbuildersettings.md:35]: Alright. Should we link off to some docs about how to do this, you think?
Also, I remember config transforms being a Cloud thing - but it's also used for CMS now?

**Pull request merged**